components/Comment/CommentEditor.tsx

```diff
@@ -45,41 +45,44 @@ const { setMessage, showMessage } = MessageActions;
 
 type CommentEditorArgs = {
   editorId: string;
+  handleSubmit: Function;
+  handleCancel?: Function;
   commentId?: ID;
   placeholder?: string;
-  handleSubmit: Function;
   content?: object;
   allowBounty?: boolean;
   commentType?: COMMENT_TYPES;
   author?: AuthorProfile | null;
-  previewModeAsDefault?: boolean;
+  minimalMode?: boolean;
```

Comment:
Minimal mode is essentially just showing the textbox without buttons until the user clicks into